tools/barcode_collapse/BarcodeCollapse_pe.xml

```diff
@@ -0,0 +1,48 @@
+<tool id="barcode_collapse" name="barcode_collapse" version="0.1.0.0">
+    <description>Paired End randomer aware duplicate removal algorithm !</description>
+    <requirements>
+        <requirement type="package" version="0.1.0.0">barcode_collapse</requirement>
```

Comment:
If it's just one python file that you need you can include it next to your xml file and in this PR, given that the licence allows it.